### PR TITLE
negar/reduce_delay

### PR DESCRIPTION
**Pull request by negarn**
reduce delay

### CODE REVIEW

File: src/root_files/app/index.html [hunk 1]
```diff
@@ -49,14 +48,6 @@
...
-            function isDelayedRedirect() {
-                if(isGtmApplicable() && !redirectTimeOut) {
-                    redirectTimeOut = setTimeout(redirect, 5000);
-                } else {
-                    clearTimeout(redirectTimeOut);
-                    redirect();
-                }
-            }
             function isGtmApplicable() {
                 return /(www|staging)\.binary\.com/i.test(window.location.hostname);
```
Comment (ashkanx) [inline src/root_files/app/index.html]: How about `binary.me` and `desktop_app` handling?

Comment (negarn) [reply src/root_files/app/index.html]: desktop_app has the same URL right? I'll just add the handler for binary.me

File: src/root_files/app_2/index.html [hunk 2]
```diff
@@ -49,54 +49,40 @@
...
-                if(isGtmApplicable() && !redirectTimeOut) {
-                    redirectTimeOut = setTimeout(redirect, 5000);
-                } else {
-                    clearTimeout(redirectTimeOut);
-                    redirect();
-                }
-            }
             function isGtmApplicable() {
-                return /www\.binary\.com/i.test(window.location.hostname);
+                return /(www|staging)\.binary\.(com|me)/i.test(window.location.hostname);
```
Comment (ashkanx) [inline src/root_files/app_2/index.html]: I think we don't need `isGtmApplicable` and ` pushEvents` as we don't wait for GTM anymore

Comment (negarn) [reply src/root_files/app_2/index.html]: so do we want to remove GTM from app_2/index.html completely?

**Pull request merged**